mobile/lib/src/infrastructure/eeg_data_transformers/ganglionTransformer.dart

```diff
@@ -1,67 +1,36 @@
-import 'dart:io';
-
-import 'package:csv/csv.dart';
-import 'package:path_provider/path_provider.dart';
-import 'package:polydodo/src/domain/eeg_data/eeg_data.dart';
-import 'package:polydodo/src/domain/eeg_data/i_eeg_data_repository.dart';
-import 'package:polydodo/src/domain/unique_id.dart';
-import 'constants.dart';
-
-class EEGDataRepository implements IEEGDataRepository {
-  bool _streamInitialized = false;
-  EEGData _recordingData;
-  List _lastSampleData = [0, 0, 0, 0, 0];
+import 'package:polydodo/src/infrastructure/constants.dart';
+import 'baseTransformer.dart';
+
+class GanglionTransformer<S, T> extends BaseTransformer<S, T> {
+  List _lastSampleData;
   int _sampleCounter;
 
-  void createRecordingFromStream(Stream<List<int>> stream) {
-    _recordingData =
-        EEGData(UniqueId.from(DateTime.now().toString()), List<List>());
+  GanglionTransformer.broadcast({bool synchronous: false, cancelOnError})
```

Comment:
why redefine it?

Reply:
En fait, si je le redéfini pas, il me dit qu'un constructeur .broadcast n'existe pas pour Ganglion et Cyton. J'ai cherché un peu et j'ai pas trouvé pourquoi exactement. Il ne semble pas voir le constructeur .broadcast de la classe de base. Je sais pas si c'est à cause de mon implémentation ou c'est fait comme ça.